ClassAttendance/src/BLL/Interfaces/ILecturerService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BLL.Models;
+
+namespace BLL.Interfaces
+{
+    public interface ILecturerService : IService<Person>
```

Comment:
I guess we decided to remove these interfaces.

Reply:
I have to inject 2 different implementations of IService<Person>, because i have one implementation for studens and one for lecturers. So that's the only way i found to let DI know which implementation i want use

Reply:
It's not a problem and can be registered without additional interfaces.
IService<Person> - it should be requested in the constructor when you try to use it.

Reply:
I guess - the problem is a Person model.
You should have 2 different models for Student and Lecturer.
It's norman if it will almost or even equal models.
But we should have different models due to the difference in business meaning.

Reply:
For sure problem is in model.
When i started this task i had 2 different models, then i asked u about best way to deal with 2 same models and u adviced me to create 1 model with field that will help to undestand which one model it is. 
Please let me know is there any ways to resolve one of implementations of IService<Person> or i will just split it to different models.

Reply:
Okay, let's create 2 different models.